kimyeongseok/app.js

```diff
@@ -0,0 +1,147 @@
+const http = require ("http");
+const express = require ("express");
+const cors = require ("cors");
+const morgan = require ("morgan");
+
+const dotenv = require ("dotenv");
+dotenv.config()
+
+const { DataSource } = require('typeorm');
+const { json } = require("express");
+
+const mysqlDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE  
+})
+
+mysqlDataSource.initialize()
+ .then(() => {
+    console.log("Data Source has been initialized")
+ })
+
+app = express()
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('dev'))
+ 
+app.get ("/ping", (req,res) => {
+    res.status(200).json({ message : "pong"});
+})
+
+app.post("/users", async (req,res)=>{
+  const { name, profile_image, password, email } = req.body
```

Comment:
클라이언트에게 받는 데이터 변수명은 JavaScript의 언어 컨벤션 중 하나인 camel case를 따를 수 있도록 수정해주세요!
```suggestion
  const { name, profileImage, password, email } = req.body
```